Application/EdFi.Ods.Admin.Api/Features/ClaimSets/DeleteClaimSetModelValidator.cs

```diff
@@ -28,10 +21,6 @@ public DeleteClaimSetModelValidator(IGetClaimSetByIdQuery getClaimSetByIdQuery)
             RuleFor(m => m.Id).NotEmpty()
                 .Must(BeAnExistingClaimSet)
                 .WithMessage("No such claim set exists in the database");
-            RuleFor(m => m.VendorApplicationCount)
-                .Equal(0)
-                .WithMessage(m => $"Cannot delete this claim set. This claim set has {m.VendorApplicationCount} associated application(s).");
-            RuleFor(m => m.IsEditable).NotEqual(false).WithMessage("Only user created claim sets can be deleted");
         }
```

Comment:
`VendorApplicationCount` does not exist. Does this mean that we are missing an important validation?